.changes/next-release/feature-a75d9a1e-37d0-4d9e-a13d-6e6eb12d0edd.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Add a AWS Explorer ECR node"
```

Comment:
an*

Reply:
I think this is actually grammatically correct since it's `a...node` but an sounds better, changed